src/test/java/co/omise/live/LiveScheduleRequestTest.java

```diff
@@ -24,7 +26,7 @@ public void setup() throws Exception {
     }
 
     @Test
-    @Ignore("only hit the network when we need to.")
+    //@Ignore("only hit the network when we need to.")
```

Comment:
Live tests supposed to be run on local only (at least for now).

Reply:
Yeah forget to uncomment.  Fixed now